Lib/pathlib.py

```diff
@@ -1222,23 +1222,18 @@ def touch(self, mode=0o666, exist_ok=True):
     def mkdir(self, mode=0o777, parents=False, exist_ok=False):
```

Comment:
Could you quickly add a docstring? Maybe "Create a new directory at this given path."?

A more thorough one is probably even better explaining the parameters, but the one-liner is better than nothing.